modules/network-observability-flowmetrics-charts.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-custom-charts-flowmetrics_{context}"]
+= Configuring custom charts using FlowMetrics API
+You can generate charts for dashboards in the {product-title} web console, which you can view as an administrator in the *Dashboard* menu by defining the `charts` section of the `FlowMetric` resource.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetrics*.
+. Use the *Project:*  dropdown list to select the project of the Network Observability Operator instance. 
+. Click *Create FlowMetric*.
+. Here are some examples for configuring the `FlowMetric` resource: 
+
+.Chart for tracking ingress bytes received from cluster external sources
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-traffic
+  namespace: netobserv   <1>
+# ...
+  charts:
+  - dashboardName: Main  <2>
+    title: External ingress traffic
+    unit: Bps
+    type: SingleStat
+    queries:
+    - promQL: "sum(rate($METRIC[2m]))"
+      legend: ""
+  - dashboardName: Main  <2>
+    sectionName: External
+    title: Top external ingress traffic per workload
+    unit: Bps
+    type: StackArea
+    queries:
+    - promQL: "sum(rate($METRIC{DstK8S_Namespace!=\"\"}[2m])) by (DstK8S_Namespace, DstK8S_OwnerName)"
+      legend: "{{DstK8S_Namespace}} / {{DstK8S_OwnerName}}"
+# ...
+----
+<1> The same namespace, `netobserv`, is used across all components. You can optionally use a different namespace for the different components. It is important to keep namespace names consistent across all components.
+
+.Verification
+. Once the pods refresh, navigate to *Observe* -> *Dashboards*.
+. Search for the *NetObserv / Main* dashboard.
+. View two panels under the *NetObserv / Main* dashboard, or optionally a dashboard name that you create:
```

Comment:
is this a separate step. It can be a part of previous step, IMO e.g. `You can view....`